com.ugetdm.uGet.yaml

```diff
@@ -0,0 +1,119 @@
+app-id: com.ugetdm.uGet
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: uget-gtk
+rename-icon: uget-icon
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
Yes, it's based on GTK3

Reply:
given the upstream desktop file that seem to want to force X11, I'll ask again. Did you actually test it on Wayland (XWayland doesn't count) ?